services/imageproxy/local.go

```diff
@@ -66,6 +67,20 @@ func makeLocalBackend(proxy *ImageProxy) *LocalBackend {
 	}
 }
 
+type contentTypeRecorder struct {
+	http.ResponseWriter
+	filename string
+}
+
+func (rec *contentTypeRecorder) WriteHeader(code int) {
+	hdr := rec.ResponseWriter.Header()
+	if hdr.Get("Content-Type") == "image/svg+xml" {
```

Comment:
Matching directly against the header value won't work since it could contain additional params. Parse with [`mime.ParseMediaType`](https://golang.org/pkg/mime/#ParseMediaType) first.

Reply:
done